js/web/lib/wasm/jsep/webnn/tensor-manager.ts

```diff
@@ -54,6 +54,33 @@ export interface TensorManager {
 let tensorGuid = 1;
 const createNewTensorId = (): TensorId => tensorGuid++;
 
+/**
+ * Map from MLOperandDataType to size in bytes.
+ */
+const webnnDataTypeToSize = new Map<MLOperandDataType, number>([
+  ['float32', 4],
+  ['float16', 2],
+  ['int32', 4],
+  ['uint32', 4],
+  ['int64', 8],
+  ['uint64', 8],
+  ['int8', 1],
+  ['uint8', 1],
+  ['int4', 0.5],
+  ['uint4', 0.5],
```

Comment:
Hmm, rather than floats (which have precision issues around 16777216 elements), can we use integers like bits and divide by /8?

```
...
  ['uint4', 4],
...
  (a * b * bitSize) / 8
...
```

Reply:
Done